service.conf.example

```diff
@@ -0,0 +1,11 @@
+# enable debug logging
+YUBIKEY_TOUCH_DETECTOR_VERBOSE=false
+
+# show desktop notifications using libnotify
+YUBIKEY_TOUCH_DETECTOR_LIBNOTIFY=true
+
+# print notifications to stdout
+YUBIKEY_TOUCH_DETECTOR_STDOUT=false
+
+# disable Un*x socket notifier
+YUBIKEY_TOUCH_DETECTOR_NOSOCKET=false
```

Comment:
If you feel like it, could I ask you also to update https://github.com/maximbaz/yubikey-touch-detector#configuring-the-app with the missing config values? :pray: